src/Mvc/Collection/Behavior.php

```diff
@@ -0,0 +1,88 @@
+<?php declare(strict_types=1);
+
+/**
+ * This file is part of the Phalcon Framework.
+ *
+ * (c) Phalcon Team <team@phalconphp.com>
```

Comment:
this is a minor fix
Please replace the email with team@phalcon.io
of course not only in this file
Thanks!

Reply:
Ok, working

Reply:
Its okay